doc/user.rst

```diff
@@ -62,27 +62,26 @@ In any case, you need:
 Each plugin usually needs a codec library, which you also need to
 install. Check the :doc:`plugins` for details about required libraries
 
-For example, the following installs a fairly complete list of build dependencies on Debian Jessie:
+For example, the following installs a fairly complete list of build dependencies on Debian Buster:
 
 .. code-block:: none
 
-    apt install g++ \
+    apt install meson g++ \
       libpcre3-dev \
       libmad0-dev libmpg123-dev libid3tag0-dev \
-      libflac-dev libvorbis-dev libopus-dev \
+      libflac-dev libvorbis-dev libopus-dev libogg-dev \
       libadplug-dev libaudiofile-dev libsndfile1-dev libfaad-dev \
-      libfluidsynth-dev libgme-dev libmikmod2-dev libmodplug-dev \
+      libfluidsynth-dev libgme-dev libmikmod-dev libmodplug-dev \
       libmpcdec-dev libwavpack-dev libwildmidi-dev \
-      libsidplay2-dev libsidutils-dev libresid-builder-dev \
+      libsidplayfp-dev libsidutils-dev libresid-builder-dev \
```

Comment:
Why switch from libsidplay to libsidplayfp? Both are included in Buster.

Reply:
I don't know, [Florian switched to libsidplayfp in 2016](https://salsa.debian.org/mpd-team/mpd/commit/34bef5eb) but no motivation is included in the changelog. Do you recommend libsidplay over libsidplayfp?

Reply:
I use neither, I don't have any SID files (except for MPD development), I have no idea which library fork is better. But your commit was about changes for Buster, and this is not one.
If somebody argues that the MPD documentation should suggest one or the other, that would be okay for me, but this way, it's hidden inside an unrelated commit, and I don't like that.
Maybe we should just remove this library from here, because I think only very few users are interested in SID playback.